docs/reference/opentext_documentum.md

```diff
@@ -0,0 +1,121 @@
+
+## Availability and prerequisites
+
+This connector is available as a self-managed connector client.
+To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is a **tech preview connector** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties.
+
+## Usage
+
+To use this connector as a **connector client**, select the **Opentext Documentum** tile when creating a new connector under **Search -> Connectors**.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+## Connecting to Opentext Documentum
+
+Username and password based authentication is used to connect with Opentext Documentum.
+
+Once completed, use the following parameters to configure the connector.
+
+## Configuration
+
+### Configure Opentext Documentum connector
+
+Note the following configuration fields:
+
+#### `Opentext Documentum host url`  (required)
+
+The domain where Opentext Documentum is hosted. Example:
+
+- `https://192.158.1.38:2099/`
+
+#### `Username`  (required)
+
+The username of the account to connect with Opentext Documentum.
+
+#### `Password`  (required)
+
+The password of the account to connect with Opentext Documentum.
+
+#### `Repositories`
+
+Comma-separated list of repositories to fetch data from Opentext Documentum. If the value is `*` the connector will fetch data from all repositories present in the configured user’s account.
+
+Default value is `*`.
+
+Examples:
+
+- `elastic`, `kibana`
+- `*`
+
+#### `Enable SSL`
+
+Enable SSL for the Opentext Documentum instance.
+
+#### `SSL Certificate`
+
+SSL certificate for the Opentext Documentum instance. Example:
+
+```
+-----BEGIN CERTIFICATE-----
+MIID+jCCAuKgAwIBAgIGAJJMzlxLMA0GCSqGSIb3DQEBCwUAMHoxCzAJBgNVBAYT
+...
+7RhLQyWn2u00L7/9Omw=
+-----END CERTIFICATE-----
+```
+
+### Content Extraction
+
+Refer to [content extraction](https://www.elastic.co/guide/en/enterprise-search/current/connectors-content-extraction.html) in the official docs.
+
+## Documents and syncs
+
+The connector syncs the following objects and entities:
+- **Repositories**
+- **Cabinets**
+- **Files & Folders**
+
+*NOTE*:
+- Files bigger than 10 MB won't be extracted.
+- Permissions are not synced. **All documents** indexed to an Elastic deployment will be visible to **all users with access** to the destination Elasticsearch index.
+
+### Sync types
+[Full syncs](https://www.elastic.co/guide/en/enterprise-search/current/connectors-sync-types.html#connectors-sync-types-full) are supported by default for all connectors.
+
+This connector currently does not support [incremental syncs](https://www.elastic.co/guide/en/enterprise-search/current/connectors-sync-types.html#connectors-sync-types-incremental).
+
+## Sync rules
+
```

Comment:
Advanced sync rules?

Reply:
This connector does not have advanced sync rules.

Reply:
Cool, let's make that explicit in docs so :)

Reply:
Done, added.